pkg/proto/nbcontract/v1/tlsbootstrappingconfig.proto

```diff
@@ -1,12 +1,9 @@
 syntax = "proto3";
 package nbcontract.v1;
 import "pkg/proto/nbcontract/v1/featurestate.proto";
-
+ 
 message TLSBootstrappingConfig {
```

Comment:
looks like you forgot to checkout master before creating your new branch :P

Reply:
oops good catch, fixed